ui/components/src/TimeSeriesTooltip/utils.ts

```diff
@@ -40,17 +45,12 @@ export function assembleTransform(
   const x = mousePos.page.x;
   let y = mousePos.page.y + cursorPaddingY;
 
-  const isCloseToBottom = mousePos.page.y > window.innerHeight * 0.8;
-  const yPosAdjustThreshold = chartHeight * 0.75;
-  // adjust so tooltip does not get cut off at bottom of chart, reduce multiplier to move up
-  if (isCloseToBottom === true) {
-    if (seriesNum > 6) {
-      y = mousePos.page.y * 0.75;
-    } else {
-      y = mousePos.page.y * 0.9;
-    }
-  } else if (mousePos.plotCanvas.y > yPosAdjustThreshold) {
-    y = mousePos.page.y * 0.95;
+  // approximate tootlip height based on number of series
+  const tooltipHeight = Math.min(Math.min(seriesNum, TOOLTIP_MAX_ITEMS) * 24, TOOLTIP_MAX_HEIGHT);
```

Comment:
Awesome, much better than the static `if (seriesNum > 6)` hack I did before, plus this actually fixes the core problem!

Reply:
nit: typo in comment that says `tootlip` instead of `tooltip`